packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -475,6 +474,10 @@ export class Cluster extends ClusterBase {
       encrypted: props.encrypted !== undefined ? props.encrypted : true,
     });
 
+    if (clusterType === ClusterType.MULTI_NODE) {
+      cluster.numberOfNodes = nodeCount;
+    }
```

Comment:
Instead of setting this after the creation, it would be better to set the value of `nodeCount` correctly before passing it to `CfnCluster`.

This should also throw an error if a user specifies `numberOfNodes` on a `SINGLE_NODE` cluster.